### PR TITLE
Adapting call to sdc-client after sdc-client interfce change

### PR DESCRIPTION
**Pull request by jmmovilla**
***** REVIEWERS *****
@jesuspg

### CODE REVIEW

**Comment by jmmovilla** [comment]
Please, do not merge so far

**Comment by jframos** [comment]
Travis error:

*[ERROR] COMPILATION ERROR :*
*[ERROR] /home/travis/build/telefonicaid/fiware-paas/core/src/main/java/com/telefonica/euro_iaas/paasmanager/installator/ProductInstallatorSdcImpl.java:[177,102] error: incompatible types*

**Comment by jframos** [comment]
We must merge this PR ASAP because the related one to this issue in the SDCManager component has already been merged. If not, we will have integration problems...

File: core/pom.xml [hunk 1]
```diff
@@ -60,7 +60,7 @@
         <dependency>
             <groupId>com.telefonica.iot</groupId>
             <artifactId>sdc-client</artifactId>
-            <version>2.6.0</version>
+            <version>2.8.0-SNAPSHOT</version>
```
Comment (jesuspg) [inline core/pom.xml]: last version for SDC is 2.9.0-SNAPSHOT: https://github.com/telefonicaid/fiware-sdc/blob/develop/pom.xml#L9

**Comment by jesuspg** [comment]
LGTM

**Comment by jframos** [comment]
LGTM

**Pull request merged**